### DISCUSSION TITLE
Resources and Experienced help

### DISCUSSION THREAD

**Discussion by ghost** [community/community#145930] [Other Feature Feedback, Questions, & Ideas]
Guys, I'm kind of new to this site so theres a few things I got to work on it so excuse me if it's off. I personally want too and need to grow my educational resources as far as people who are experinced! I am fairly new to the computer science career. As far as that goes, I am working toward a few difference things school eventually! As of now though, If anyone is willing to add me on discord or Tele so I can reach out for any questions regaurding any topics or maybe find a group to coloberate with on here for all purposes

**Comment by GabJuarez** [comment]
Hey, I’m new too! I’m just starting out in computer science as well, so I totally get where you're coming from. It’s great to connect with others who are in the same boat, and we can definitely learn and grow together. If you ever have any questions or want to collaborate, feel free to reach out! : )

**Comment by ghost** [comment]
What are you working towards are working on currently?

**Reply by GabJuarez** [reply]
I’m working on my final course project, called Hospital Management, Im coding it in C#

**Comment by herman16** [comment]
please help me i  just want to ask if github can develop my program made in visual basic 6.0 on the website. Thank you very much for answering my question. ang how much to develop my program?

**Comment by ghost** [comment]
I am not quite sure on that but I have found Visual Studio Code is pretty good on my part, but on the other behalf I've never honestly made or developed anything quite yet slow but eventually

**Comment by shinybrightstar** [comment]
Welcome to GitHub and the GitHub Community @EdwardIrvin96  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
